app/src/org/odk/collect/android/jr/extensions/ImageRestrictionExtensionParser.java

```diff
@@ -0,0 +1,28 @@
+package org.odk.collect.android.jr.extensions;
+
+import org.javarosa.core.model.QuestionDataExtension;
+import org.javarosa.xform.parse.QuestionExtensionParser;
+import org.javarosa.xform.parse.XFormParser;
+import org.kxml2.kdom.Element;
+
+/**
+ * Created by amstone326 on 8/20/15.
+ */
+public class ImageRestrictionExtensionParser extends QuestionExtensionParser {
+
+    public ImageRestrictionExtensionParser(String elementName) {
+        super(elementName);
+    }
+
+    @Override
+    public QuestionDataExtension parse(Element e) {
+        String s = e.getAttributeValue(XFormParser.NAMESPACE_JAVAROSA,
+                "imageDimensionScaledMax");
+        if (s != null) {
+            // Parse off the "px" and cast to int
+            int maxDimens = Integer.parseInt(s.substring(0, s.length() - 2));
```

Comment:
What will happen if the pixel limit is bad (either short or not an int)? it looks like we'll hard crash - maybe we should just log and not set a pixel limit?

Also: I think I think in other places (like the case list) we allow 100px and just 100 - maybe we should do that here too? (IE just remove the "px" if it exists)